q2_composition/assets/run_ancombc.R

```diff
@@ -69,8 +69,9 @@ output_loaf         <- opt$output_loaf
 if (!file.exists(inp_abundances_path)) {
   errQuit("Input file path does not exist.")
 } else {
-  otu_file <- t(read.delim(inp_abundances_path, check.names = FALSE,
-                            row.names = 1))
+  # This is the old code, which should fail with the new test data
+  otu_file <- t(read.delim(inp_abundances_path, check.names = FALSE, # TRUE,
+                            row.names = 1)) #, colClasses = c(X = "character")))
```

Comment:
Once we have a unit test that fails to import table-ancom-IDE.qza,
we can fix this function with
`check.names = TRUE`
`row.names = 1, colClasses = c(X = "character")))`